### PR TITLE
Fix formatting and grammar issues in :doc:`/contributing/development`

### PR DESCRIPTION
**Pull request by bolded**
## Overview: What does this pull request change?

- Removed a reference to a non-existent (404) flowchart at the bottom of the page
- Changed list element to be more consistent
- Capitalized Manim where necessary
- Fixed grammar referring to Graphviz dependency

## Links to added or changed documentation pages

https://manimce--2904.org.readthedocs.build/en/2904/contributing/development.html

## Further Information and Comments

If an image is available for the flowchart, it would be best to update it instead of removing the reference altogether. It seems that it may have been a 404 for a long time.

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Review by sparshg — APPROVED** [review]
Looks good, thanks!

File: docs/source/contributing/development.rst [hunk 1]
```diff
@@ -88,11 +90,11 @@ For first-time contributors
 
       poetry run pre-commit install
 
-   This will ensure during development that each of your commits is properly
+   This will ensure during development that each of your commits are properly
```
Comment (behackl) [inline docs/source/contributing/development.rst:93]: each ... is, or alternatively something like all ... are, but I don't agree with this proposed change as is.

**Review by behackl — CHANGES_REQUESTED** [review]
LGTM, except for one particular proposed change. In case I don't hear back, I'll change it and merge this then. Sorry for the long delay!

**Review by behackl — APPROVED** [review]
Resolved conflict and commited suggested change. Ready to be merged, thanks for your contribution!

**Pull request merged**